include/aws/common/common.h

```diff
@@ -117,12 +117,12 @@
  * Violations of the function contracts are undefined behaviour.
  */
 #ifdef CBMC
-#define AWS_PRECONDITION(cond) __CPROVER_assume(cond)
+#define AWS_PRECONDITION(cond, explanation) __CPROVER_precondition((cond), (explanation))
 #define AWS_POSTCONDITION(cond) AWS_ASSERT(cond)
```

Comment:
Since we are using `__CPROVER_precondition` for `AWS_PRECONDITION`, should we also use `__CPROVER_assert` for `AWS_POSTCONDITION`? @danielsn @angelhof

Reply:
Note that `__CPROVER_assert` requires a second argument (an explanation).